.github/workflows/ubuntu_20.04/build-deps.sh

```diff
@@ -125,4 +125,25 @@ wget -q https://github.com/rouault/pdfium_build_gdal_3_4/releases/download/v2_pd
   && apt-get install -y --fix-missing --no-install-recommends liblcms2-dev \
   && rm -rf /var/lib/apt/lists/*
 
+# HANA: client side
+# Install hdbsql tool
+curl -v -j -k -s -L -H "Cookie: eula_3_1_agreed=tools.hana.ondemand.com/developer-license-3_1.txt" https://tools.hana.ondemand.com/additional/hanaclient-latest-linux-x64.tar.gz --output hanaclient-latest-linux-x64.tar.gz \
+  && tar -xvf hanaclient-latest-linux-x64.tar.gz \
+  && mkdir /usr/sap \
+  && ./client/hdbinst -a client --sapmnt=/usr/sap \
+  && rm -rf client \
+  && rm hanaclient*
+export PATH=/usr/sap/hdbclient:$PATH
+
+wget -q https://github.com/Kitware/CMake/releases/download/v3.12.4/cmake-3.12.4-Linux-x86_64.sh -O cmake.sh \
```

Comment:
ubuntu 20.04 comes with cmake 3.16 and it is already installed by previous steps

Reply:
Thank you. Removed in 4ed4eb3ab2e6510e1f98be48f0d26a4ea345c488.